book/src/keymap.md

```diff
@@ -400,19 +400,21 @@ you to selectively add search terms to your selections.
 
 Keys to use within picker. Remapping currently not supported.
 
-| Key                          | Description       |
-| -----                        | -------------     |
-| `Shift-Tab`, `Up`, `Ctrl-p`  | Previous entry    |
-| `Tab`, `Down`, `Ctrl-n`      | Next entry        |
-| `PageUp`, `Ctrl-u`           | Page up           |
-| `PageDown`, `Ctrl-d`         | Page down         |
-| `Home`                       | Go to first entry |
-| `End`                        | Go to last entry  |
-| `Enter`                      | Open selected     |
-| `Ctrl-s`                     | Open horizontally |
-| `Ctrl-v`                     | Open vertically   |
-| `Ctrl-t`                     | Toggle preview    |
-| `Escape`, `Ctrl-c`           | Close picker      |
+| Key                          | Description                              |
+| -----                        | -------------                            |
+| `Shift-Tab`, `Up`, `Ctrl-p`  | Previous entry                           |
+| `Tab`, `Down`, `Ctrl-n`      | Next entry                               |
+| `PageUp`, `Ctrl-u`           | Page up                                  |
+| `PageDown`, `Ctrl-d`         | Page down                                |
+| `Home`                       | Go to first entry                        |
+| `End`                        | Go to last entry                         |
+| `Enter`                      | Open selected                            |
+| `Ctrl-s`                     | Open horizontally                        |
+| `Ctrl-v`                     | Open vertically                          |
+| `Ctrl-t`                     | Toggle preview                           |
+| `Escape`, `Ctrl-c`           | Close picker                             |
+| `Ctrl-l`                     | Move file picker to parent directory     |
+| `Ctrl-o`                     | Narrow file picker on selected directory |
```

Comment:
The prior art is emacs's helm-mode, which uses `ctrl-l` for a similar purpose, and I somewhat arbitrarily picked `ctrl-o` because it was nearby and not taken.

I would be happy to change these commands to something more intuitive. Do you think `ctrl-o` to "move out" and `ctrl-i` to "move in" would be the best?

Reply:
Ah, we cannot use `ctrl-i` as the terminal sends that as `tab`, which is already bound.

Instead, I have opted for `ctrl-e` and `ctrl-a` as these already have meaning for going forward and backward, though I'm happy to change to something else.